### PR TITLE
Upgrade web-animations-js to v2.2.0

### LINKED ISSUE #162: Error: Failed to execute 'animate' on 'Element': 'function (a){return a}' is not a valid value for easing
### Description
Getting console error when using neon-animations with a build of Chrome that throws TypeErrors on invalid timing functions.
`Error: Failed to execute 'animate' on 'Element': 'function (a){return a}' is not a valid value for easing`
This is due to a bug in the web-animations-js polyfill that has been fixed in v2.2.0.

### Expected outcome
Repo tests should pass.

### Actual outcome
Most fail with the above error.

### Live Demo
Needs a custom build of Chrome with https://codereview.chromium.org/1851003002 patched in.

### Steps to reproduce
1. Build Chrome with https://codereview.chromium.org/1851003002.
2. Run neon-animation tests.

### Browsers Affected

- [ ] Chrome
- [ x ] Future Chrome
- [ ] Firefox
- [ x ] Firefox Nightly
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

### PR DESCRIPTION
**Pull request by alancutter**
Fixes #162.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "iron-meta": "PolymerElements/iron-meta#^1.0.0",
     "iron-resizable-behavior": "PolymerElements/iron-resizable-behavior#^1.0.0",
     "iron-selector": "PolymerElements/iron-selector#^1.0.0",
-    "web-animations-js": "web-animations/web-animations-js#2.1.4"
+    "web-animations-js": "web-animations/web-animations-js#2.2.0"
```
Comment (dstockwell) [inline bower.json]: This should probably be `^2.2.0`, no reason to force to a particular version.

Comment (alancutter) [inline bower.json]: Done.

**Comment by samuelli** [comment]
lgtm

**Comment by alancutter** [comment]
@dstockwell is OOO at the moment, would you be able to review this change @cdata ?

**Pull request merged**